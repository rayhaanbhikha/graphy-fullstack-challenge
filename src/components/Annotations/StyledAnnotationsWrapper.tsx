import styled from 'styled-components'

export const StyledAnnotationsWrapper = styled.div`
  position: inherit;
  height: inherit;
  width: inherit;
`