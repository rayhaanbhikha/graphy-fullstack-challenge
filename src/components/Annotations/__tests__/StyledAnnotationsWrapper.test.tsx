import React from 'react';
import { render } from '@testing-library/react'
import { StyledAnnotationsWrapper } from '../StyledAnnotationsWrapper';
import 'jest-styled-components';

describe('StyledAnnotationsWrapper', () => {

  it('should have the default styles', () => {
    const { asFragment } = render(<StyledAnnotationsWrapper />);
    expect(asFragment()).toMatchSnapshot();
  })
});