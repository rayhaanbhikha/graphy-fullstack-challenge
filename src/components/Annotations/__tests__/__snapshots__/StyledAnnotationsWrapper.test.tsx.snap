// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledAnnotationsWrapper should have the default styles 1`] = `
<DocumentFragment>
  .c0 {
  position: inherit;
  height: inherit;
  width: inherit;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
